### PR TITLE
support for `query_settings` for clickhouse models

### PR DESCRIPTION
**Pull request by k-anshul**

### CODE REVIEW

File: runtime/drivers/object_store.go [hunk 1]
```diff
@@ -42,6 +42,6 @@ type ObjectStoreModelOutputProperties struct {
 }
 
 type ObjectStoreModelResultProperties struct {
-	Path   string     `mapstructure:"path"`
-	Format FileFormat `mapstructure:"format"`
+	Path   string `mapstructure:"path"`
+	Format string `mapstructure:"format"`
```
Comment (begelundmuller) [inline runtime/drivers/object_store.go:46]: Why this change?

Comment (k-anshul) [reply runtime/drivers/object_store.go:46]: When we are storing the results in catalog here : https://github.com/rilldata/rill/blob/b970338f4df225c7fc3d1acb86c27872ae52fe5c/runtime/reconcilers/model.go#L523-L524
it is not able to handle `FileFormat` type
It needs go default types only.

Comment (k-anshul) [reply runtime/drivers/object_store.go:46]: I think we do not hit this case right now but we will hit this case if we just have a model which has `snowflake` as input and `s3` as output.

Comment (begelundmuller) [reply runtime/drivers/object_store.go:46]: Got it

**Review by begelundmuller — APPROVED** [review]
